ui/Decoder.elm

```diff
@@ -13,7 +13,7 @@ buildRecordDecoder =
         |> required "build" buildDecoder
         |> required "download" downloadDecoder
         |> required "source" sourceDecoder
-        |> required "systemaddons" (nullable (list systemAddonDecoder))
+        |> required "systemaddons" (maybe (list systemAddonDecoder))
```

Comment:
But the patch currently does that already. Switching to `nullable` keeps the error occuring even with the decoder properly updated.